packages/ckb-sdk-rpc/types/CKB_RPC/index.d.ts

```diff
@@ -30,11 +30,16 @@ declare module CKB_RPC {
     code_hash: Hash256
   }
 
-  export interface OutPoint {
+  export interface CellOutPoint {
     tx_hash: Hash256
     index: Index
   }
 
+  export interface OutPoint {
```

Comment:
Does this allow parsing JSON with both `cell` and ` block_hash` as `null`?

The genesis block's inputs would have inputs like that. Will this stop us from parsing that JSON?

Reply:
allowed block hash and cell to be null in https://github.com/nervosnetwork/ckb-sdk-js/pull/161/commits/3885fab9f3cf1b49e9f4ada6b27c2f9e73d99644